### DISCUSSION TITLE
Module not found while using bcrypt into Next.js v14

### DISCUSSION THREAD

**Discussion by imashmishra** [kelektiv/node.bcrypt.js#1034] [Q&A]
![Screenshot 2024-05-01 132056](https://github.com/kelektiv/node.bcrypt.js/assets/113676063/34579b0c-3646-481e-abce-b8a3ba62259b)
![Screenshot 2024-05-01 132104](https://github.com/kelektiv/node.bcrypt.js/assets/113676063/87d49d8d-da04-4a78-b8d4-f521c2a17344)
I am getting this issue while starting my project via npm run dev, don't know why stuck into this for the last 2 days if anyone is up for help, I have tried all the things like uninstalling the node_module and reinstalling it, but all these things are not work for me

**Comment by ezeparziale** [comment]
Check https://github.com/kelektiv/node.bcrypt.js/issues/979

**Reply by imashmishra** [reply]
Thanks for the help. The issue was resolved when I replaced bcrypt with bcryptjs

**Comment by BertramYe** [comment]
but what if I use the turbo compiler in the Nextjs 15 rc , how can I resolve this issue?

**Comment by imashmishra** [comment]
Hey everyone, if you’re looking for an answer, I resolved my issue by replacing bcrypt with bcryptjs. Try this if you’re encountering the same problem.
